buildpack/telemetry/metrics.py

```diff
@@ -128,6 +128,18 @@ def get_appmetrics_target():
     return os.getenv("APPMETRICS_TARGET")
 
 
+def get_micrometer_metrics_url():
```

Comment:
Few points:
- `USE_TRENDS_FORWARDER` as a feature flag is good, but we need to switch to `TRENDS_FORWARDER_URL` only if it is not empty.
- Can we add some comments or docstring to explain why we have a separate function as opposed to `get_metrics_url` below?
- How about we call this method `get_metrics_url` instead of having reference to the micrometer and have the other one be called `get_trends_storage_url` so it's unambiguous?
- Can we add a unit-test for this method or update any existing ones?

Reply:
@pappachino Another suggestion: we can leave `get_metrics_url` only, but add a parameter `micrometer_only=True', and call it this way for Telegraf

Reply:
* Added check for TRENDS_FORWARDER_URL 
* Docstring has been added
* Decided to keep names the same
* Unit test added